conjur/controller/login_controller.py

```diff
@@ -58,7 +59,7 @@ def get_username(self):
             self.credential_data.login = input("Enter your username: ").strip()
             if self.credential_data.login == '':
                 # pylint: disable=raise-missing-from
-                raise RuntimeError("Error: Login name is required")
+                raise MissingRequiredParameterException("Error: Login name is required")
```

Comment:
Notice that we have `MissingParametersException` and `MissingRequiredParameterException`. Why do we need both?

Reply:
MissingParametersException removed